mlir/utils/jenkins/miopen-tests/miopen_validate.sh

```diff
@@ -133,7 +134,13 @@ function run_tests() {
     clean_miopen_caches
 
     local exit_status
-    parallel <"$TMPFILE"
+    # The extra bash -c accounts for parallel trying to pass each line as one
+    # command-line argument
+    # The 'awk' invocation prints all lines of output while ensuring indicators of
+    # success are present
+    parallel bash -c {} '|' awk \
+    "'BEGIN { status=2 } /Verifies OK/ { status=status-1 } /ConvMlirIgemm/ { status=status-1 } 1; END { exit(status) }'" \
```

Comment:
[Nit] This will do the job, but the return code calculation seems less intuitive then a `if FAILED then exit(1) fi` calculation. I don't have a strong opinion on this.

Reply:
I'm here going for "succeed if both conditions are met", while also printing the output.